css/styles.css

```diff
@@ -0,0 +1,100 @@
+body {
+  background-color: #f0f0f0;
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 500;
+}
+
+header {
+  /* Header background */
+  background-color: #562b81;
+}
+
+.page-header {
+  color: #fff;
+  font-size: 1.125rem;
+}
+
+h1,
+h2 {
+  font-family: 'Proza Libre', sans-serif;
+}
+
+h2 {
+  font-weight: 100;
+}
+
+h1 {
+  font-weight: 600;
+}
+
+label {
+  display: block;
+}
+
+img {
+  width: 100%;
+  margin-bottom: 0.6rem;
+  border-radius: 0.4rem;
+  padding-left: 0%;
+}
+
+.center-row {
+  display: flex;
+  flex-direction: row;
```

Comment:
These are already being applied to your `row`, so you don't need to repeat this.